src/modules/layout/components/Footer.js

```diff
@@ -1,30 +1,53 @@
 // @flow
 
+import type { Node } from 'react'
```

Comment:
I prefere to use React.Node. As this does not cause confustion with the ES Node class.